browser-test/src/applicant/questions/id.test.ts

```diff
@@ -184,4 +177,60 @@ test.describe('Id question for applicant flow', () => {
       await validateAccessibility(page)
     })
   })
+
+  test.describe('single id question with north star flag enabled', () => {
+    const programName = 'Test program for single id'
+
+    test.beforeAll(async () => {
+      await setUpForSingleInput(programName)
+    })
+
+    test.beforeEach(async () => {
+      const {page} = ctx
+      await enableFeatureFlag(page, 'north_star_applicant_ui')
+    })
+
+    test(
```

Comment:
question (non-blocking):  I noticed that you aren't testing all the conditions that are tested with the north star feature flag off.  Is the idea that once the north star is in production, we'll reuse those tests against the north star code?  If so, how do we know they'll all pass with these changes?

Reply:
That makes sense.  I do like the idea of breaking things up into smaller PRs.